pkg/machinery/constants/constants.go

```diff
@@ -814,6 +814,16 @@ var Overlays = []string{
 	"/etc/cni",
 	"/etc/kubernetes",
 	"/usr/libexec/kubernetes",
-	"/usr/etc/udev",
 	"/opt",
 }
+
+// DefaultDroppedCapabilities is the default set of capabilities to drop.
```

Comment:
importing `cap` package breaks darwin builds of talosctl, so using the string representation instead